soroban-env-host/src/host.rs

```diff
@@ -2031,8 +2031,12 @@ impl VmCallerEnv for Host {
             .borrow_mut()
             .touch_key(&key, self.as_budget())?;
 
-        let min_expiration =
-            self.with_ledger_info(|li| Ok(li.sequence_number.saturating_add(min.into())))?;
+        let min_expiration = self.with_ledger_info(|li| {
+            Ok(li
+                .sequence_number
+                .saturating_sub(1)
```

Comment:
Does allowance.rs need to be updated as well? I think `(expiration - ledger_seq)` should become `(expiration - ledger_seq + 1)` right?

Reply:
Good catch, updated.